docs/pages/server-access/getting-started.mdx

```diff
@@ -0,0 +1,384 @@
+---
+title: Server Access Getting Started Guide
+description: Getting started with Teleport Server Access
+---
+
+# Getting Started
+
+Teleport Server Access provides full SSH capabilities out-of-the-box:
+
+1. Teleport's `tsh` combines support for SSH and SCP in one convenient local command-line tool.
+2. Each Teleport node can be configured into SSH mode (Teleport Node) and run as an enhanced SSH server.
+
+The steps below involve:
+
+1. Creating a Teleport Auth node cluster we'll add a Teleport SSH Node to.
+2. Using `tsh` to SSH into the cluster without external SSH dependencies.
+3. Introspecting the cluster using advanced `tsh` features.
+
+## Prerequisites
+
+- A Linux environment (such as Digital Ocean droplets, Azure Linux VM's, or AWS EC2) within which we'll create multiple instances.
+- A registered Doman Name.
+- A Two-Factor Authentication app (such as Authy or Google Authenticator).
+- Teleport installed locally (for `tsh`).
+- A GitHub user belonging to an organization and a team within that organization.
+
+## Step 1a. Create a cluster
+
+1. Verify you have a registered domain name. 
+
+   i. We'll create two subdomains, one for each Linux instance: `auth.example.com` and `ssh.example.com`.
+
+   ![Subdomains](../../img/server-access/subdomains.png)
+
+   ii. Then, we'll associate TLS (SSL) certificates from Let's Encrypt with them below.
+
+   iii. Make sure ports: `9090`, `443`, `3023-3025`, and `3080` are all open.
+
+2. Create two new instances of your desired Linux distribution (such as CentOS, Debian, Ubuntu).
```

Comment:
We should explain what those instances will be used for: `tele.example.com` is our bastion server hosting proxy and auth, and the other server is just a node we are connecting to the proxy.